.changeset/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.3/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    {
+      "repo": "@forgerock/forgerock-javascript-sdk"
+    }
+  ],
+  "commit": false,
+  "fixed": [["@forgerock/javascript-sdk", "@forgerock/ping-protect"]],
```

Comment:
Is there a reason Token Vault is not included here?

Reply:
I didnt want to fix token vault since we arent really releasing it for now. But we can easily add it later on if we wanted